packages/datadog-plugin-router/src/index.js

```diff
@@ -29,8 +29,9 @@ class RouterPlugin extends WebPlugin {
         context.middleware.push(span)
       }
 
-      this._storeStack.push(storage.getStore())
-      this.enter(span)
+      const store = storage.getStore()
+      this._storeStack.push(store)
+      this.enter(span, store)
```

Comment:
This part is unnecessary as the `store` parameter defaults to the value of `storage.getStore()` when omitted.

Reply:
This is just an optimization to avoid calling `storage.getStore()` twice. All other plugins appear to do the same (this was the only call to `Plugin.enter` with a single parameter).

Reply:
This is just an optimization to avoid calling `storage.getStore()` twice. All other plugins appear to do the same (this was the only call to `Plugin.enter` with a single parameter).

Reply:
> This part is unnecessary as the `store` parameter defaults to the value of `storage.getStore()` when omitted.

This is just an optimization to avoid calling `storage.getStore()` twice. All other plugins appear to do the same (this was the only call to `Plugin.enter` with a single parameter).